queries.sql

```diff
@@ -37,13 +37,8 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT name FROM animals 
 WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
 
-SELECT species, 
-MIN(weight_kg) AS min_weight, 
-MAX(weight_kg) AS max_weight 
-FROM animals
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals
 GROUP BY species;
 
-SELECT species, 
-AVG(escape_attempts)
-FROM animals
+SELECT species, AVG(escape_attempts) FROM animals
```

Comment:
- [ ] This query also returns the wrong data. This is because you forgot to include a check for animals born between `1990` and `2000`. Kindly include this check and you would get the right result.

#### Hint: You want to check using the `date_of_birth` column between `'1990/01/01'` and `'2000/12/31'` 👍